manifests/charts/base/files/profile-compatibility-version-1.21.yaml

```diff
@@ -0,0 +1,3 @@
+pilot:
+  env:
+    ENABLE_RESOLUTION_NONE_TARGET_PORT: "false"
```

Comment:
This feels like a loaded question but if we are putting this into the 1.21 compat version, shouldn't we also add it to the 1.20 compat version we already have?

Reply:
Yes! I thought I did this but got mixed up in my PRs :face_exhaling: (I have 2 PRs out adding things to compat versions).

Maybe a test for this would be handy

Reply:
Oh I forgot to `git push`...